package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ghosler-cli",
-  "version": "1.0.84",
+  "version": "1.0.86",
   "description": "A basic, simple CLI for installing, managing a Ghosler instance.",
```

Comment:
should be `managing Ghosler instances` now that we support managing multiple instances.

Reply:
Done